packages/fetchai/protocols/tac/message.py

```diff
@@ -162,10 +170,9 @@ def good_id_to_name(self) -> Dict[str, str]:
         return cast(Dict[str, str], self.get("good_id_to_name"))
 
     @property
-    def info(self) -> Dict[str, str]:
+    def info(self) -> Optional[Dict[str, str]]:
         """Get the 'info' content from the message."""
-        assert self.is_set("info"), "'info' content is not set."
```

Comment:
the generater generates 167, which should not be the case as the content type is optional